### PR TITLE
Add vite-imagetools to loader list

### PR DESCRIPTION
**Pull request by JonasKruckenberg**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described.
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list.
- [x] Your plugin/tool is working with Vite 2.0 and onward.

#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [ x The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

#### `Apps/Websites`

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vite, provided that guests could reasonably check out how Vite was used A commercial product using Vite, provided that guests could reasonably check out how Vite was used (i.e. A headless CMS which uses Vite for the Admin/editor Area and offers a free tier).

### CODE REVIEW

**Comment by antfu** [comment]
Is it framework agnostic?

**Comment by JonasKruckenberg** [comment]
Not quite yet, it relies on vites asset loader at the moment.
I have some plans to change that but I don't know when

File: README.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-plugin-vuedoc](https://github.com/JasKang/vite-plugin-vuedoc) - Markdown Code block as Vue Preview components.
 - [vite-plugin-md](https://github.com/antfu/vite-plugin-md) - Markdown as Vue components / Vue components in Markdown.
 - [vite-svg-loader](https://github.com/jpkleemans/vite-svg-loader) - Load SVG files as Vue components.
+- [vite-imagetools](https://github.com/JonasKruckenberg/vite-imagetools) - Load and transform Images using a modular workflow.
```
Comment (antfu) [inline README.md:155]:
```suggestion
- [vite-imagetools](https://github.com/JonasKruckenberg/vite-imagetools) - Load and transform images using url query.
```

**Review by antfu — CHANGES_REQUESTED** [review]
Please move to the "Framework-agnostic Plugins" section.

File: README.md [hunk 2]
```diff
@@ -118,6 +118,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [vite-plugin-rsw](https://github.com/lencx/vite-plugin-rsw) - Load rust-compiled(wasm-pack) WebAssembly packages.
 - [vite-plugin-fonts](https://github.com/stafyniaksacha/vite-plugin-fonts) - Webfont loader.
+- [vite-imagetools](https://github.com/JonasKruckenberg/vite-imagetools) - Load and transform Images using a modular workflow.
```
Comment (antfu) [inline README.md:121]:
```suggestion
- [vite-imagetools](https://github.com/JonasKruckenberg/vite-imagetools) - Load and transform images using a modular workflow.
```

**Comment by JonasKruckenberg** [comment]
Sorry @antfu didn't see the request,  but should all be good now right?

**Pull request merged**